contracts/mock/UnipoolMock.sol

```diff
@@ -4,8 +4,6 @@ import "../../contracts/Unipool.sol";
 
 contract UnipoolMock is Unipool {
 
-    constructor(IERC20 _uniswapTokenExchange, IERC20 _tradedToken) Unipool(_uniswapTokenExchange) public {
-        uniswapTokenExchange = _uniswapTokenExchange;
-        tradedToken = _tradedToken;
+    constructor(IUniswapV2Pair _uniswapTokenExchange, IERC20 _tradedToken, IUniswapV2Router01 _uniswapRouter) Unipool(IERC20(address(_uniswapTokenExchange)), _uniswapRouter, _tradedToken) public {
     }
 }
```

Comment:
You can delete this mock and reference the Unipool contract directly now that params are passed in the constructor.